drivechain-server/server/server.go

```diff
@@ -24,12 +26,45 @@ import (
 var _ rpc.DrivechainServiceHandler = new(Server)
 
 func New(wallet *bdk.Wallet, bitcoind *coreproxy.Bitcoind) *Server {
-	return &Server{wallet, bitcoind}
+	s := &Server{wallet: wallet, bitcoind: bitcoind}
+	go s.balanceUpdateLoop(context.Background())
```

Comment:
I think I might prefer this to be an exported method on `Server`. That way we can pass in a context we can control the lifecycle of